server/models/UserSkills.model.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  AllowNull,
+  Column,
+  CreatedAt,
+  ForeignKey,
+  Model,
+  Table,
+  UpdatedAt,
+} from 'sequelize-typescript';
+
+interface User_SkillsAttributes {
+  user_id: number;
+  skill_id: number;
+  created_at: Date;
+  updated_at: Date;
+}
+
+import Skills from './skills.model';
+import User from './User.model';
+
+@Table({
+  tableName: 'User_Skills',
+  timestamps: true,
+})
+
+class User_Skills extends Model<User_SkillsAttributes> implements User_SkillsAttributes {
```

Comment:
models should be singular because they describe one thing 
tables should be plural because they house multiple of the same type of things

so for the `User_Skills` model, change it to `User_Skill`
same goes for the file name